autoemulate/experimental/emulators/base.py

```diff
@@ -107,7 +108,10 @@ class PyTorchBackend(nn.Module, Emulator, Preprocessor):
     verbose: bool = False
     preprocessor: Preprocessor | None = None
     loss_fn: nn.Module = nn.MSELoss()
+    optimizer_cls: type[optim.Optimizer] = optim.Adam
     optimizer: optim.Optimizer
+    lr: float = 1e-1
+    scheduler: ExponentialLR | None = None
```

Comment:
Maybe we don't need a `None` option here, but I guess it gives the option to not use a scheduler if that was wanted

Reply:
Can we do a base scheduler class here?